chimney/src/main/scala-3/io/scalaland/chimney/compiletime/DefinitionsPlatform.scala

```diff
@@ -0,0 +1,150 @@
+package io.scalaland.chimney.internal.compiletime
+
+import io.scalaland.chimney.dsl as dsls
+import io.scalaland.chimney.internal
+import io.scalaland.chimney.{partial, PartialTransformer, Patcher, Transformer}
+
+import scala.quoted
+
+private[compiletime] trait DefinitionsPlatform(using val quotes: quoted.Quotes)
```

Comment:
`using quotes: quoted.Quotes` not enough? why do you need val?

Reply:
Without `val` you would not see this `quotes` in mixins and you would not be able to import it.